shared/src/business/entities/cases/Case.setCaseCaption.test.ts

```diff
@@ -17,4 +17,34 @@ describe('setCaseCaption', () => {
 
     expect(updatedCase.caseCaption).toEqual('A whole new caption');
   });
+
+  it('should set the case caption on the consolidated case array', () => {
```

Comment:
This test asserts that `setCaseCaptionInConsolidatedCases` sets the case caption for a _single_ case in a consolidated group. The `it` declaration should better reflect that.